sensapex/sensapex.py

```diff
@@ -753,15 +760,19 @@ def recv_all(self):
     def _update_moves(self):
         with self.lock:
             for dev, move in list(self._last_move.items()):
-                if move.is_in_progress():
-                    continue
-                if move.has_more_calls_to_make():
-                    move.make_next_call()
-                elif move.can_retry() and not move.is_close_enough():
-                    move.retry()
-                else:
-                    self._last_move.pop(dev)
-                    move.finish()
+                try:
+                    if move.is_in_progress():
+                        continue
+                    if move.has_more_calls_to_make():
+                        move.make_next_call()
+                    elif move.can_retry() and not move.is_close_enough():
+                        move.retry()
+                    else:
+                        self._last_move.pop(dev)
+                        move.finish()
+                except Exception as exc:
+                    print(f"Error processing move on sensapex device {dev}")
+                    sys.excepthook(*sys.exc_info())
```

Comment:
Not a criticism of this PR, but I wish we had a better thing to do with these exceptions; someone, somewhere would like to know this move failed and why so that they can fix their call, and it's unfortunate that we can't better tie the error to the invocation that brought us here. Maybe we can print all that out? E.g. `f"Error while attempting to move {dev} from {orig_pos} with args: {move_args}"` Or even print out the stack at the point where the move request is made?

The only call to `_update_moves` is already wrapped in this same try/except pattern, so it feels like we're maybe doing double duty on one of these, but I can't figure out how to cleanly unify them, so I'll just note as much for posterity.